filebeat/module/kafka/_meta/config.yml

```diff
@@ -0,0 +1,13 @@
+- module: kafka
+  # All logs
+  log:
+    enabled: true
+
+    # Set custom paths for Kafka. If left empty,
+    # Filebeat will look under /opt.
+    #var.kafka_home:
+
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
```

Comment:
This comment is out-of-date now, I think. It should say it chooses the paths based on the `kibana_path` value.